Wyvarn/calamus
# Allow a user to create a Blog

## Description

Create a blog from request payload. This will allow a user with a Writer role to create a blog

Fixes #28 

## How can we test this?

1. Run unit tests with `yarn test` or `npm run test`
2. Run `docker-compose up` to run Mongo DB
3. Run `yarn start` or `npm run start`
4. Make a curl request like below: (modify fields where possible):

``` bash
curl --request POST \
  --url http://localhost:3000/api/v1/writer/blog \
  --header 'authorization: Bearer <TOKEN>' \
  --header 'content-type: application/json' \
  --data '{
	"title": "Introduction to Node.js and How Node.js works?",
	"description": "Node.js is a popular framework for building backend systems (network applications) which can scale very well. Node.js is a Javascript runtime i.e. it runs the javascript codes.",
	"text": "<p>This will depend on how you render the blog. It can be html text or plain text.</p>",
	"slug": "introduction-to-node-js-and-how-node-js-works",
	"imgUrl": "https://s3.ap-south-1.amazonaws.com/afteracademy-server-uploads/introduction-to-node-js-how-node-js-works-cover.jpg",
	"tags": [
		"NODEJS",
		"BACKEND"
	]
}'
```
> Ensure the token is the same from the login endpoint as specified in the API Blueprint

You should be able to have create a blog

## Checklist:
